.github/workflows/test.yaml

```diff
@@ -43,9 +43,14 @@ jobs:
     defaults:
       run:
         working-directory: ${{ env.COMPONENT_NAME }}
+    strategy:
+      matrix:
+        instance:
+          - defaults
+          - empty
     steps:
       - uses: actions/checkout@v2
         with:
           path: ${{ env.COMPONENT_NAME }}
       - name: Golden diff
-        run: make golden-diff
+        run: make golden-diff -e instance=${{ matrix.instance }}
```

Comment:
I think you need to add the matrix to `.sync.yml`, otherwise the next modulesync PR is going to revert this.
See https://github.com/projectsyn/modulesync-control/blob/master/moduleroot/.github/workflows/test.yaml.erb
```yaml
.github/workflows/test.yaml:
  matrix:
    key: instance
    entries:
      - defaults
      - empty
```
However, current modulesync template doesn't support `make golden-diff -e instance=${{ matrix.instance }}` yet, so we need to first add something to the template.

Reply:
Ah right I didn't think of that. I'll probably make PR for that afterwards then.